src/ape/managers/chain.py

```diff
@@ -504,6 +505,44 @@ def __getitem__(self, address: AddressType) -> ContractType:
 
         return contract_type
 
+    def get_multiple(
```

Comment:
Could combine this with `get()` but that makes the `default=` argument kind of strange so a separate method made more sense.

Reply:
Let's just go with it for now.